implementations/rust/ockam/ockam_api/src/cli_state/notifications.rs

```diff
@@ -0,0 +1,2 @@
+// 00: consider making this an enum w/ hints about which messages to log (all messages can be printed to stderr or spinner by default)
```

Comment:
I think // 00: is meant to be a // TODO: ... I use this in a vscode extension that I have :)